dockerfiles/buster/Dockerfile.saithrift-server

```diff
@@ -49,18 +49,23 @@ RUN apt-get install -y thrift-compiler libthrift-dev libthrift-0.11.0 \
     && pip3 install ctypesgen thrift==0.11.0 ;
 
 ENV SAITHRIFTV2=y
-ENV GEN_SAIRPC_OPTS="-cve"
+ENV GEN_SAIRPC_OPTS="-ve"
 ENV SAIRPC_EXTRA_LIBS="-L/usr/local/lib/ -lpthread"
 
+COPY scripts/meta/* /scripts/meta/
```

Comment:
Hmm... Probably we can come up with some logic that validates `SAI_ID` and clone another SAI refpoint in case saithriftv2 should be updated. E.g., we can get commit date in UNIX timestamp format: `git show -s --format=%ct {commit id}`... or just check for a specific content in saithriftv2 to decide whether it should be updated